chia-wallet/src/proof.rs

```diff
@@ -0,0 +1,40 @@
+use clvm_traits::{FromClvm, Result, ToClvm};
+use clvmr::{allocator::NodePtr, Allocator};
+
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub enum Proof {
```

Comment:
I was hoping this would implement `Arbitrary` too, to get some round-trip coverage of `from_clvm` and `to_clvm`